test/e2e/actions/networkpeer/gcp.go

```diff
@@ -0,0 +1,53 @@
+package networkpeer
+
+import (
+	"context"
+
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/api/gcp"
+
+	"google.golang.org/api/compute/v1"
+)
+
+func CreateVPC(gcpProjectID string, vnetName string) error {
+	computeService, err := compute.NewService(context.Background())
+	if err != nil {
+		return err
+	}
+	networkService := compute.NewNetworksService(computeService)
+	net := &compute.Network{
+		Name:                  vnetName,
+		AutoCreateSubnetworks: false,
+		// make sure AutoCreateSubnetworks field is included in request
+		ForceSendFields: []string{"AutoCreateSubnetworks"},
+	}
+	_, err = networkService.Insert(gcpProjectID, net).Do()
+	return err
+}
+
+func EstablishPeerConnectionWithVPC(gpcProjectID, vnetName, atlasGCPProjectID, atlasVnetName string) error {
+	computeService, err := compute.NewService(context.Background())
+	if err != nil {
+		return err
+	}
+	networkService := compute.NewNetworksService(computeService)
+	peer := &compute.NetworkPeering{
+		Name:                 "peer",
+		Network:              gcp.FormNetworkURL(atlasVnetName, atlasGCPProjectID),
+		ExchangeSubnetRoutes: true,
+	}
+	request := &compute.NetworksAddPeeringRequest{
+		NetworkPeering: peer,
+	}
+	_, err = networkService.AddPeering(gpcProjectID, vnetName, request).Do()
+	return err
+}
+
+func DeleteGCPvpc(gcpProjectID, vnetName string) error {
```

Comment:
This function name seems strange as `vpc` is lowercased. Also might be a good idea to have some standard naming for these functions for search purposes (like `DeleteVPCforAWS`, `DeleteVPCforAzure`, `DeleteVPCforGCP`). Not necessary ofc but what do you think?

Reply:
Done